kong/plugins/aws-lambda/request-util.lua

```diff
@@ -0,0 +1,22 @@
+local ERR = ngx.ERR
+
+return function(config)
+  ngx.req.read_body()
+  local body = ngx.req.get_body_data()
+
+  if not body then
+    -- see if body was buffered to tmp file, payload could have exceeded client_body_buffer_size
+    local body_filepath = ngx.req.get_body_file()
+    if body_filepath then
+      if config.skip_large_bodies then
```

Comment:
instead of passing the entire conf table, can we just pass the option?

Reply:
yep, good call